cfgov/unprocessed/css/cf-enhancements.less

```diff
@@ -520,6 +520,29 @@ tbody {
 
 
 /* topdoc
+  name: Hide
+  family: cf-core
+  patterns:
+    - name: Utility class
+      markup: |
+        <h1 class="u-hide">
+          Show using JavaScript
+        </h1>
+      codenotes:
+        - .u-hide;
+      notes:
+        - "Use this class to hide something,
+          perhaps to be shown using JavaScript."
+  tags:
+    - cf-core
+*/
+
+.u-hide {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/edaeaa74656fba1628f0b9706b2178d9f4a49fed

Also removed some redundant changelog entries I found.